db/blob/blob_source.cc

```diff
@@ -81,7 +82,23 @@ Status BlobSource::PutBlobIntoCache(const Slice& cache_key,
 
 Cache::Handle* BlobSource::GetEntryFromCache(const Slice& key) const {
   Cache::Handle* cache_handle = nullptr;
-  cache_handle = blob_cache_->Lookup(key, statistics_);
+
+  if (lowest_used_cache_tier_ == CacheTier::kNonVolatileBlockTier) {
+    Cache::CreateCallback create_cb = [&](const void* buf, size_t size,
+                                          void** out_obj,
+                                          size_t* charge) -> Status {
+      *out_obj = reinterpret_cast<void*>(new char[size]);
```

Comment:
`char*` is implicitly convertible to `void*`, so no need to cast

Reply:
Thanks! @ltamasi

Reply:
Done.